librz/analysis/rzil/rzil_stats.c

```diff
@@ -0,0 +1,89 @@
+// SPDX-FileCopyrightText: 2021 heersin <teablearcher@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_analysis.h>
+
+static int hook_flag_read(RzAnalysisRzil *rzil, const char *flag, ut64 *num, RzAnalysis *analysis) {
```

Comment:
I wonder if it's better to move `RzAnalysis *analysis` to the first argument instead.